examples/anthos-service-mesh-multicluster/README.md

```diff
@@ -0,0 +1,167 @@
+- [Twistlock PoC](./twistlock)    
+- [Aiven MySQL PoC](./mysql-docker)
+
+# Multi-Cluster ASM on Private Clusters
+
+## Documentation
+
+Here are several reference documents if you encounter issue when follow along instructions below:
+
+- [Installing ASM using Anthos CLI](https://cloud.google.com/service-mesh/docs/gke-anthos-cli-existing-cluster)
+- [Installing ASM using IstioCtl](https://cloud.google.com/service-mesh/docs/gke-install-existing-cluster)
+- [Adding clusters to an Athos Service Mesh](https://cloud.google.com/service-mesh/docs/gke-install-multi-cluster)
+
+## Description
+
+In [Adding clusters to an Athos Service Mesh](https://cloud.google.com/service-mesh/docs/gke-install-multi-cluster), it shows how to federate service meshes of two Anthos **public** clusters. However, it misses a key instruction to open firewall for the service port to remote cluster. So, you final test of HelloWorld might not work. 
+
+This sample builds on topic of Google's Anthos Service Mesh official installation documents, and adds instruction on how to federate two private clusters, which is more demanded in real customer environments. 
+
+As illustrated in the diagram below, we will create a VPC with three subnets. Two subnets are for private clusters, and one for GCE servers. So, we illustrate using bastion server to access private clusters as in real environment. 
+
+![NetworkImage](./asm-private-multiclusters-intranet.png)
+
+The clusters are not accessible from external network. User can only log into the bastion server via IAP tunnel to gain access to this VPC. A firewall is built to allow IAP tunnel into GCE subnet (Subnet C) only. For bastion server in Subnet C to access Kubernetes APIs of both private clusters, Subnet C CIDR range is added to the "_Master Auth Network_" of both clusters. This is illustrated as blue lines and yellow underscore lines in the diagram above.
```

Comment:
Is there a reason why we are not deploying a bastion as part of the infra config?

Reply:
Done! Good suggestion. Updated readme to reflect this